modules/nw-ingress-creating-a-route-via-an-ingress.adoc

```diff
@@ -42,9 +39,8 @@ spec:
     secretName: example-com-tls-certificate
 ----
 +
-<1> The `route.openshift.io/termination` annotation can be used to configure the `spec.tls.termination` field of the `Route`
-as `Ingress` has no field for this. The accepted values are `edge`, `passthrough` and `reencrypt`. All
-other values are silently ignored. When the annotation value is unset, `edge` is the default route. The TLS certificate details must be defined in the template file to implement the default edge route.
+<1> The `route.openshift.io/termination` annotation can be used to configure the `spec.tls.termination` field of the `Route` as `Ingress` has no field for this. The accepted values are `edge`, `passthrough` and `reencrypt`. All other values are silently ignored. When the annotation value is unset, `edge` is the default route. The TLS certificate details must be defined in the template file to implement the default edge route.
+<2> When working with an Ingress object, you must specify an explicit host name, unlike when working with routes. You can use the `<host_name>.<cluster_ingress_domain>` syntax, for example `apps.openshiftdemos.com`, to take advantage of the cluster's `*.<cluster_ingress_domain>` wildcard DNS record and serving certificate. Otherwise, you must ensure that there is a DNS record for the chosen host name.
```

Comment:
You could change "Ingress object" to "Ingress object", given that it relates to a resource object that is being referred to.

Reply:
You could potentially remove "unlike when working with routes", to minimise the first sentence in the call out.

Reply:
You could change "the cluster's `*.<cluster_ingress_domain>` wildcard DNS record and serving certificate" to something like the following, so that a possessive is not used:

> "the `*.<cluster_ingress_domain>` wildcard DNS record and serving certificate for the cluster"

Reply:
You could change "host name" to "hostname". The ISG recommends using "hostname" except when updating existing content that uses "host name".